.github/workflows/ci.yaml

```diff
@@ -140,7 +140,7 @@ jobs:
       run: make docker-build
 
     - name: Deploy controller to local cluster
-      run: make deploy
+      run: make _deploy # do not check env vars
```

Comment:
One alternative would be swapping this to `make deploy NGROK_API_KEY=fake-ci-check NGROK_AUTHTOKEN=fake-ci-check`

I think that might feel a little better since it lets us not have a ci-specific make target

Reply:
Ah, I was considering just adding the env vars on the actions job, but this feels a little better. I can do that.